src/shared/Microsoft.Git.CredentialManager.Tests/GenericHostProviderTests.cs

```diff
@@ -58,20 +58,89 @@ public void GenericProvider_GetCredentialKey_ReturnsCorrectKey()
             Assert.Equal(expectedKey, actualKey);
         }
 
+        [Fact]
+        public async Task GenericHostProvider_CreateCredentialAsync_WiaNotAllowed_ReturnsBasicCredentialNoWiaCheck()
+        {
+            var input = new InputArguments(new Dictionary<string, string>
+            {
+                ["protocol"] = "https",
+                ["host"]     = "example.com",
+            });
+
+            const string testUserName = "basicUser";
+            const string testPassword = "basicPass";
+            var basicCredential = new GitCredential(testUserName, testPassword);
+
+            var context = new TestCommandContext
```

Comment:
worth having a case without user?  That should be the more popular case correct?

Reply:
We can add a case, yes.

> That should be the more popular case correct?

Actually no. When GCM is installed (by default with Git for Windows) it will be getting of all basic prompts first (not Git). I imagine there are more users who use basic authentication rather than those who use it as a workaround for a PAT.